interfaces/seccomp/template.go

```diff
@@ -218,8 +218,8 @@ fadvise64
 fadvise64_64
 arm_fadvise64_64
 
-mbarrier
 mbind
+membarrier
```

Comment:
This is reverted in https://github.com/ubuntu-core/snappy/pull/1088/commits/67a820685d57e0927d1c9ce745e8086d92cf355f